docs/source/how-to/customize-initialization.rst

```diff
@@ -93,6 +94,9 @@ Preloads can also be registered with configuration at the following values:
      nanny:
        preload: []
        preload_argv: []
```

Comment:
I have a similar comment here about the indention level for the `nanny` block and using `preload-argv` instead of `preload_argv`. Could you update this section too? (I realize this predates the changes in this PR, but we should clean it up while we're making changes anyways)